src/test/versionHandlers.test.ts

```diff
@@ -0,0 +1,96 @@
+import { updateTrajectoryFileInfoFormat } from "../simularium/versionHandlers";
+
+const invalidVersionData = {
+    connId: "7496831076a233f0-2c337fed-4493-ad92-79f194744174ba05635426fd",
```

Comment:
couldn't this just have `version: 999.9` and no other data? seems like it would be a clearer test case that way,

Reply:
I may be mis-remembering my TS, but wouldn't any type-checks related to the `updateTrajectoryFileInfoFormat` function called fail without the other data?

Reply:
Yes @EricJIsaac is correct -- the test works with just the version number, but TS needs everything else 😛